### PR TITLE
Implemented sorting algorithms

### PR DESCRIPTION
**Pull request by philips-ts**

### CODE REVIEW

File: src/main/java/com/tarasenko/sorting_algorithms/CombSort.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+public class CombSort {
+
+    public void sort(int[] array) {
+        int gap = array.length;
+        boolean swapped = true;
+        while (gap > 1 || swapped) {
+            if (gap > 1) {
+                gap = (int) (gap / 1.247330950103979);
+            }
```
Comment (ALnOUA) [inline src/main/java/com/tarasenko/sorting_algorithms/CombSort.java]: It`s better to use static variable with magic numbers

File: src/main/java/com/tarasenko/sorting_algorithms/InsertionSort.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+    public static void main(String[] args) {
+        InsertionSort insertionSort = new InsertionSort();
+        int[] array = new int[10];
+        Random random = new Random();
+
+        for (int i = 0; i < array.length; i++) {
+            array[i] = random.nextInt(100);
+        }
```
Comment (ALnOUA) [inline src/main/java/com/tarasenko/sorting_algorithms/InsertionSort.java]: the same

File: src/main/java/com/tarasenko/sorting_algorithms/InsertionSort.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            }
+            array[i + 1] = value;
+        }
+    }
+
+
+    public static void main(String[] args) {
+        InsertionSort insertionSort = new InsertionSort();
+        int[] array = new int[10];
+        Random random = new Random();
```
Comment (ALnOUA) [inline src/main/java/com/tarasenko/sorting_algorithms/InsertionSort.java]: the same

File: src/main/java/com/tarasenko/sorting_algorithms/CountingSort.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        }
+    }
+
+    public static void main(String[] args) {
+        CountingSort countingSort = new CountingSort();
+        int[] array = new int[10];
+        Random random = new Random();
+
+        int maxNumber = 100;
+        for (int i = 0; i < array.length; i++) {
```
Comment (ALnOUA) [inline src/main/java/com/tarasenko/sorting_algorithms/CountingSort.java]: Better to declare it as a class static variable
public static final int ARRAY_MAX_NUMBER = 100;
or something like that

**Review by ALnOUA — CHANGES_REQUESTED** [review]
Fix similar comments all over the code

File: src/main/java/com/tarasenko/sorting_algorithms/CombSort.java (hunk 1 above)
Comment (philips-ts) [reply src/main/java/com/tarasenko/sorting_algorithms/CombSort.java]: Done

File: src/main/java/com/tarasenko/sorting_algorithms/InsertionSort.java (hunk 2 above)
Comment (philips-ts) [reply src/main/java/com/tarasenko/sorting_algorithms/InsertionSort.java]: Done

File: src/main/java/com/tarasenko/sorting_algorithms/CountingSort.java (hunk 4 above)
Comment (philips-ts) [reply src/main/java/com/tarasenko/sorting_algorithms/CountingSort.java]: Done

**Review by ALnOUA — APPROVED** [review]

**Pull request merged**